skimage/morphology/tests/test_flood_fill.py

```diff
@@ -112,6 +112,48 @@ def test_inplace_noncontiguous():
     np.testing.assert_allclose(image, expected)
 
 
+def test_inplace_int_deprecated():
+    """This test is deprecated and will be removed in
+    version 0.19.0. See #4248.
+    """
+    image = np.array([[0, 0, 0, 0, 0, 0, 0],
+                      [0, 1, 1, 0, 2, 2, 0],
+                      [0, 1, 1, 0, 2, 2, 0],
+                      [1, 0, 0, 0, 0, 0, 3],
+                      [0, 1, 1, 1, 3, 3, 4]])
+
+    flood_fill(image, (0, 0), 5, inplace=True)
```

Comment:
```suggestion
    with expected_warnings(['The inplace']):
        flood_fill(image, (0, 0), 5, inplace=True)
```
Needs a 
```python
from skimage._shared.testing import expected_warnings
```
at the top